app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,3 @@
+class SubscriptionsController < ApplicationController
```

Comment:
You can delete this - the subscriptions are all controlled via the nested attributes.